### ISSUE TITLE
OLH Contact form required indicator missing style

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5115]
<img width="309" height="368" alt="Image" src="https://github.com/user-attachments/assets/81dc30cb-4163-46f3-bcfa-d9e0eec23c10" />

Looks like the style for the key  (a red asterisk) is not the same as used in the form (a black asterisk).

**Comment by mauromsl** [comment]
closed by #5534
